homeassistant/components/mqtt/sensor.py

```diff
@@ -63,6 +65,8 @@
 DEFAULT_NAME = "MQTT Sensor"
 DEFAULT_FORCE_UPDATE = False
 
+URL_DOCS_SUPPORTED_SENSOR_UOM = "https://developers.home-assistant.io/docs/core/entity/sensor/#available-device-classes"
```

Comment:
Should we link https://www.home-assistant.io/integrations/sensor/#device-class instead?